pkg/kubelet/apis/podresources/types.go

```diff
@@ -51,3 +52,8 @@ type MemoryProvider interface {
 	// GetAllocatableMemory returns the allocatable memory from the node
 	GetAllocatableMemory() []*podresourcesapi.ContainerMemory
 }
+
+type DRAProvider interface {
```

Comment:
We've been avoiding using `DRA` as an acronym in public APIs (even in internal code). Can you change this to `DynamicResourcesProvider`? It seems to be more consistent with the rest of the naming here anyway.